common/lib/xmodule/xmodule/tabs.py

```diff
@@ -390,6 +390,19 @@ def iterate_displayable(course, user=None, inline_collections=True):
                     yield tab
 
     @classmethod
+    def swap_tabs(cls, tabs):
```

Comment:
Nit: I'd call this ``upgrade_tabs`` or something that implies that it can do anything to change the set of tabs for legacy courses. The method as it stands doesn't just swap the first two tabs as it renames them, and in the future I could imagine it doing other upgrades.